samples/snippets/src/main/java/com/example/spanner/PgAsyncQueryToListAsyncExample.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
```

Comment:
Would it make more sense to place all the PG samples in a separate sub-package. So something like `com.example.spanner.postgresql`? (I'm open to both solutions, both the one that is chosen now by prefixing the samples with `Pg`, as well as moving them to a separate package, but I would like us to take an informed decision on it)

Reply:
Sure, will decide in today call with @jin-jj and decide the approach.

Reply:
@olavloite : I think this was templated on https://github.com/googleapis/java-spanner/pull/1700/files.

IIRC, If we move to a sub-package - the class path would end up becoming something like `samples/snippets/src/main/java/com/example/spanner/postgres/AsyncQueryToListAsyncExample.java`, where the only gain probably will be readability of the class name, and some segregation from the regular GSQL samples.

The seggregation still happens in the current scheme of things, and looking for Spangres samples with an explicit pg in them sounds all right to me.

So we can keep things as is.

Reply:
Thanks @olavloite  and @ansh0l.  So @Sivakumar-Searce we will keep the class name as-is.